hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestDeleteWithSlowFollower.java

```diff
@@ -221,7 +223,9 @@ public void testDeleteKeyWithSlowFollower() throws Exception {
     }
     Assert.assertNotNull(follower);
     Assert.assertNotNull(leader);
-    // shutdown the slow follower
+    //ensure that the chosen follower is still a follower
+    Assert.assertTrue(ContainerTestHelper.isRatisFollower(follower, pipeline));
```

Comment:
Do we want to fail the test if `follower` is no longer a follower, or should we just ignore test result in that case?  If the latter, it may be better to change this to `Assume.assumeTrue`.

Reply:
There are no failures being induced till this check so, follower should still be a follower. I would prefer to fail the test in that case.